### ISSUE TITLE
OBS closes after a few seconds without ever having shown its main window on macOS

### ISSUE THREAD

**Issue by afh** [obsproject/obs-studio#2549]
## Expected Behavior
OBS version for macOS opens and runs.

## Current Behavior

OBS closes after a few seconds without ever having shown its main window

## Steps to Reproduce

1. Download OBS for macOS version 24.0.6
2. Install OBS by dragging to a folder residing on an FileVault enabled APFS (Case-sensitive) file system
3. Double click to open OBS
4. OBS closes after a few seconds without ever having shown its main window

### Possible Workaround
1. Open Terminal.app
2. Start OBS from the command-line
```
% open OBS.app
LSOpenURLsWithRole() failed with error -10810 for the file /Users/afh/Applications/OBS.app.
```
3. Starting the OBS application in a different manner hints at the underlying issue:
```
% ./OBS.app/Contents/MacOS/obs
This application failed to start because it could not find or load the Qt platform plugin "cocoa"
in "".
Reinstalling the application may fix this problem.
zsh: abort      ./OBS.app/Contents/MacOS/obs
```
3. To workaround the issue set the `QT_PLUGIN_PATH` environment variable
`% export QT_PLUGIN_PATH="$PWD/OBS.app/Contents/Plugins"`
4. then the OBS application can be opened and works properly using `% open OBS.app`

## Additional information

```
% sw_vers
ProductName:	Mac OS X
ProductVersion:	10.15.3
BuildVersion:	19D76
```

**Comment by afh** [comment]
Possibly related issue: #2511

**Comment by Fenrirthviti** [comment]
Fixed in #2443

**Comment by afh** [comment]
Thanks for the update!
